model/riscv_extensions.sail

```diff
@@ -11,6 +11,13 @@ scattered enum extension
 // Note, these are sorted according to the canonical ordering vaguely described
 // in the `Subset Naming Convention` section of the unprivileged spec.
 
+// Machine and Supervisor Architectures v1.11
+enum clause extension = Ext_Sx1p11
```

Comment:
There is no such thing as Sx1p11 etc. Ss1p11 and Sm1p11 are the extension names. We shouldn't be using non-existent extension names in the official model.

Reply:
Yeah true, I just thought it was probably overkill to have both (I'm not even entirely sure what it would mean to support Ss1p11 and Sm1p12 for example) so this was an attempt to combine them. I agree it's not very obvious though and unfortunate that it's not a real extension name.

Do you think we should have both separately, or use a different name to combine them?

Reply:
They should be separate, because from an ISA model reading perspective it's important to know whether something's attributed to Ss or Sm. But I think it's perfectly fine to have an invariant that the versions match between them (and expect that's an implicit assumption made by the privileged spec; it hurts my head to think about what mixing would even look like...).

Reply:
Ok that makes sense, I'll change it.